libraries/postgres/test/database_test.js

```diff
@@ -94,7 +95,7 @@ dbSuite(path.basename(__filename), function() {
   });
 
   test('setup creates JS methods that can be called', async function() {
-    await Database.upgrade({schema, runUpgrades: true, readDbUrl: this.dbUrl, writeDbUrl: this.dbUrl});
+    await Database.upgrade({schema, runUpgrades: true, readDbUrl: this.dbUrl, writeDbUrl: this.dbUrl, serviceName: 'service-1'});
```

Comment:
This is a bit awkward, though -- an upgrade doesn't happen in the context of a single service.